hooks/useUser.test.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import { getToken } from 'helpers/authentication';
+
+import useUser from './useUser';
+import { act } from 'react-dom/test-utils';
+
+jest.mock('helpers/authentication');
+
+describe('useUser', () => {
+  const testId = 'is-logged-in-test-id';
+  const UseAuthenticationComponent = () => {
+    const { user } = useUser({});
+    return <div data-test-id={testId}>{user?.isLoggedIn && 'logged in'}</div>;
+  };
+
+  const mockTokens = {
+    tokenType: '',
+    accessToken: '',
+    refreshToken: '',
+  };
+
+  beforeEach(() => {
+    getToken.mockImplementation(() => mockTokens);
+  });
+
+  describe('given the token in local storage exists', () => {
```

Comment:
Should we add more cases to test when there is no token in the storage?

Reply:
sure, this hooks is now tested with `testing-library/react-hooks` in 191115e3b3c247e8f417ba60f0c6ca950251c98d